### PR TITLE
Argo ISTIO config

### PR DESCRIPTION
**Pull request by gabrielwen**
Fixes kubeflow/kubeflow#2965

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/38)

### CODE REVIEW

File: argo/base/virtual-service.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  labels:
+    ksonnet.io/component: argo
```
Comment (swiftdiaries) [inline argo/base/virtual-service.yaml]: Could you please remove the ksonnet label?

**Comment by swiftdiaries** [comment]
Other changes lgtm

**Comment by gabrielwen** [comment]
please note I don't have chance to test this myself - kfctl apply doesn't work for now.

File: argo/base/virtual-service.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: argo-ui
+  namespace: kubeflow
```
Comment (hougangliu) [inline argo/base/virtual-service.yaml]: we'd better remove namespace, here

Comment (gabrielwen) [reply argo/base/virtual-service.yaml]: removed.

**Pull request merged**